### DISCUSSION TITLE
Display watch streak message text

### DISCUSSION THREAD

**Discussion by gempir** [Chatterino/chatterino2#5002] [Ideas] [answered]
Twitch has watch streaks now which are sent as USERNOTICE and do show up in chatterino, but they can have text.

Example:

```
@flags=;color=#00FF80;subscriber=1;badges=vip/1,subscriber/36,premium/1;system-msg=gempir\swatched\s3\sconsecutive\sstreams\sthis\smonth\sand\ssparked\sa\swatch\sstreak!;msg-param-value=3;login=gempir;room-id=70811818;tmi-sent-ts=1701715717735;emotes=;msg-param-id=f73739ac-135c-4dde-a80b-433f2c036fa0;badge-info=subscriber/40;mod=0;msg-id=viewermilestone;msg-param-category=watch-streak;user-id=77829817;id=bddf3269-2367-4842-ac5d-cc16ee856292;msg-param-copoReward=350;user-type=;display-name=gempir;vip=1 :tmi.twitch.tv USERNOTICE #yabbe Aware
```

How it shows up in Chatterino:
![EnuQWez](https://github.com/Chatterino/chatterino2/assets/9765622/1d591f65-519e-4af1-bdbf-780ecb98d6a7)

How it shows up in Webchat:
![ne9Y924](https://github.com/Chatterino/chatterino2/assets/9765622/001cb6ef-22fd-47da-be73-dfb662974b35)

**Comment by zneix** [comment]
Same behaviour is shared with the Announcement messages, can look at those for implementation.

**Comment by pajlada** [comment]


**Reply by gempir** [reply]
Wrong PR?

**Reply by pajlada** [reply]
Yes :sunglasses:

**Comment by pajlada** [comment, marked as answer]
Fixed in https://github.com/Chatterino/chatterino2/pull/5029
